pandas/core/window.py

```diff
@@ -780,8 +780,41 @@ def calc(x):
 
 class _Rolling_and_Expanding(_Rolling):
 
-    _shared_docs['count'] = """%(name)s count of number of non-NaN
-    observations inside provided window."""
+    _shared_docs['count'] = dedent(r"""
+    The %(name)s sum if it is a non-Nan value inside the window.
+
+    Returns
+    -------
+    Returns the object type with the sum of the window values
+    in the current row.
+
+    See Also
+    --------
+    Series.%(name)s : Calling object with Series data
+    DataFrame.%(name)s : Calling object with DataFrames
+
+    Examples
+    --------
+    >>> s = pd.Series([2, 3, np.nan, 'values'])
```

Comment:
It's a bit strange to give an example with mixed floats / strings (I mean, in general that's not really good practice to have data like that). I think only using floats (with nans) is fine.

Reply:
I wanted to show an example of mixed values being counted, in case a mistake is made by having different types of data.  I see what you are saying though since it's already stated in the description.  I will update to only use floats and nans.  Thanks!